cube/generic_networks/att_model.py

```diff
@@ -0,0 +1,215 @@
+import dynet as dy
```

Comment:
Please add a License Header based on Apache V2. You can use as an example, one of the other files in NLP-Cube.